rmw_zenoh_cpp/src/detail/graph_cache.cpp

```diff
@@ -836,7 +838,8 @@ rmw_ret_t GraphCache::publisher_count_matched_subscriptions(
   *subscription_count = 0;
   GraphNode::TopicMap::const_iterator topic_it = graph_topics_.find(publisher->topic_name);
   if (topic_it != graph_topics_.end()) {
-    rmw_publisher_data_t * pub_data = static_cast<rmw_publisher_data_t *>(publisher->data);
+    rmw_zenoh_cpp::rmw_publisher_data_t * pub_data =
+      static_cast<rmw_zenoh_cpp::rmw_publisher_data_t *>(publisher->data);
```

Comment:
Since we are already in the namespace, I don't think we need this change, right?

Reply:
Done.